### PR TITLE
feat!: adding records entity for transparency + tileFormatOutput

### PR DESCRIPTION
**Pull request by ronenkapelian**
* transperancy records (transperant or opaque) -> profile for pycsw 
* tileFormatOutput - png or jpeg for raster internal uses of new ingestion

### CODE REVIEW

File: src/models/layerMetadata/layerMetadata.ts [hunk 1]
```diff
@@ -1064,6 +1064,51 @@ export class LayerMetadata implements ILayerMetadata, IMetadataCommonModel {
   //#endregion
   public displayPath: string | undefined = undefined;
 
+  //#region RASTER: transparency
+  @pycsw({
+    profile: 'mc_raster',
+    xmlElement: 'mc:transparency',
+    queryableField: 'mc:transparency',
+    pycswField: 'pycsw:Transparency',
```
Comment (asafMasa) [inline src/models/layerMetadata/layerMetadata.ts:1072]: why do you use capital "T" letter?

File: src/models/layerMetadata/layerMetadata.ts [hunk 2]
```diff
@@ -1064,6 +1064,51 @@ export class LayerMetadata implements ILayerMetadata, IMetadataCommonModel {
...
+    column: {
+      name: 'tile_output_format',
+      type: 'text',
+      nullable: false,
+    },
+  })
+  @tsTypes({
+    mappingType: TsTypes.TILEOUTPUTFORMAT,
+  })
+
```
Comment (asafMasa) [inline src/models/layerMetadata/layerMetadata.ts:1108]: no space here

File: src/yaml/fullLayerMetadata.yaml [hunk 3]
```diff
@@ -14,3 +14,15 @@ components:
...
+              enum:
+              - TRANSPARENT
+              - OPAQUE
+              description: layer can be transparent or opaque
+            tileOutputFormat:
+              type: string
+              enum:
+              - PNG
+              - JPEG
+              description: Tile creation can be of type png or jpeg
```
Comment (asafMasa) [inline src/yaml/fullLayerMetadata.yaml:28]: Tile format can be of png or jpeg

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good job 👍

File: src/models/layerMetadata/decorators/property/tsTypes.decorator.ts [hunk 4]
```diff
@@ -94,6 +94,16 @@ export const TsTypes: Record<string, IDescribeTsType> = {
     type: PropertiesTypes.ENUM,
     importFromPackage: '@map-colonies/mc-model-types',
   },
+  TRANSPARENCY: {
+    value: 'Transparency',
+    type: PropertiesTypes.ENUM,
+    importFromPackage: '@map-colonies/mc-model-types',
+  },
+  TILEOUTPUTFORMAT: {
```
[the remark this answers is not in the document]
Comment (ronenkapelian) [reply src/models/layerMetadata/decorators/property/tsTypes.decorator.ts:102]: i saw that other types with multiple words are defined as i do without underscores

File: src/models/layerMetadata/layerMetadata.ts (hunk 1 above)
Comment (ronenkapelian) [reply src/models/layerMetadata/layerMetadata.ts:1072]: copy the convention from other examples of types

File: src/models/layerMetadata/decorators/property/tsTypes.decorator.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (ronenkapelian) [reply src/models/layerMetadata/decorators/property/tsTypes.decorator.ts:102]: changed

File: src/models/layerMetadata/layerMetadata.ts (hunk 1 above)
Comment (ronenkapelian) [reply src/models/layerMetadata/layerMetadata.ts:1072]: changed + on pycsw

File: src/yaml/fullLayerMetadata.yaml (hunk 3 above)
Comment (ronenkapelian) [reply src/yaml/fullLayerMetadata.yaml:28]: changed

File: src/models/layerMetadata/layerMetadata.ts (hunk 2 above)
Comment (ronenkapelian) [reply src/models/layerMetadata/layerMetadata.ts]: changed

**Review by netanelC — APPROVED** [review]
Amm.. Fine job.. Get H"H

**Review by RonitKissis — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

**Review by alebinson — APPROVED** [review]
Looks good!

**Pull request merged**